src/firebase/remoteConfigValuesDefaults.e2e.ts

```diff
@@ -64,4 +64,5 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   networkTimeoutSeconds: 30,
   dappFavoritesEnabled: false,
   celoNews: JSON.stringify({ enabled: false } as RemoteConfigValues['celoNews']),
+  useTwelveWordSeedPhrase: false,
```

Comment:
ah, i'm conflicted about what this value should be 😅 it's a shame we can only test one flow, but do you think we should set this to true and prioritise the 12 word seed phrase?

Reply:
As long as we are sure that the restore flow works for both either create flow seems fine for me.